test/spec/ol/webgl/helper.test.js

```diff
@@ -354,7 +358,7 @@ describe('ol.webgl.WebGLHelper', function () {
         uniform float u_test;
 
         void main(void) {
-          gl_Position = vec4(u_test, a_test, 0.0, 1.0);
+          gl_Position = vec4(u_test, attr3, 0.0, 1.0);
```

Comment:
`a_test` was an undeclared identifier in the shader used in this test.  This now throws an error.